releases/release-5.3.0.md

```diff
@@ -315,10 +316,10 @@ Starting from TiCDC v5.3.0, the cyclic replication feature between TiDB clusters
 + TiDB
 
     - Fix an error that occurs during execution caused by the wrong execution plan. The wrong execution plan is caused by the shallow copy of schema columns when pushing down the aggregation operators on partitioned tables [#27797](https://github.com/pingcap/tidb/issues/27797) [#26554](https://github.com/pingcap/tidb/issues/26554)
-    - Fix the issue that plan-cache cannot detect changes of unsigned flags [#28254](https://github.com/pingcap/tidb/issues/28254)
-    - Fix the wrong partition pruning when the partition function is out of range [#28233](https://github.com/pingcap/tidb/issues/28233)
+    - Fix the issue that `plan cache` cannot detect changes of unsigned flags [#28254](https://github.com/pingcap/tidb/issues/28254)
+    - Fix the wrong `partition pruning` when the partition function is out of range [#28233](https://github.com/pingcap/tidb/issues/28233)
```

Comment:
["Partition pruning"](https://docs.pingcap.com/zh/tidb/dev/partition-pruning) is a method of optimization. I think it's okay to leave it out of code font.